pkg/terror/error_list.go

```diff
@@ -808,6 +809,7 @@ var (
 	ErrDumpUnitRuntime        = New(codeDumpUnitRuntime, ClassDumpUnit, ScopeInternal, LevelHigh, "mydumper runs with error", "")
 	ErrDumpUnitGenTableRouter = New(codeDumpUnitGenTableRouter, ClassDumpUnit, ScopeInternal, LevelHigh, "generate table router", "Please check `routes` config in task configuration file.")
 	ErrDumpUnitGenBAList      = New(codeDumpUnitGenBAList, ClassDumpUnit, ScopeInternal, LevelHigh, "generate block allow list", "Please check the `block-allow-list` config in task configuration file.")
+	ErrDumpUnitGlobalLock     = New(codeDumpUnitGlobalLock, ClassDumpUnit, ScopeInternal, LevelHigh, "Couldn't acquire global lock", "Please check upstream privilege about FTWL, or add --no-locks to extra-args of mydumpers")
```

Comment:
my fault, it's `FTWRL`.

BTW, wrap ``` `--no-locks` ```?

Reply:
fixed in https://github.com/pingcap/dm/pull/857/commits/8c017f5c03d762303cdddef19a09496da8a01d88